spec/spdx_spec.rb

```diff
@@ -93,11 +93,11 @@
       expect(Spdx.find('lgpl_2_1').name).to eq('GNU Lesser General Public License v2.1 only')
       expect(Spdx.find('lgpl_v2_1').name).to eq('GNU Lesser General Public License v2.1 only')
 
-      expect(Spdx.find("BSD 3-Clause").name).to eq("BSD 3-clause \"New\" or \"Revised\" License")
-      expect(Spdx.find("BSD 3-clause").name).to eq("BSD 3-clause \"New\" or \"Revised\" License")
-      expect(Spdx.find("BSD 2-Clause").name).to eq("BSD 2-clause \"Simplified\" License")
-      expect(Spdx.find("BSD 2-clause").name).to eq("BSD 2-clause \"Simplified\" License")
-      expect(Spdx.find("BSD Style").name).to eq("BSD 3-clause \"New\" or \"Revised\" License")
+      expect(Spdx.find("BSD 3-Clause").name).to eq("BSD 3-Clause \"New\" or \"Revised\" License")
+      expect(Spdx.find("BSD 3-Clause").name).to eq("BSD 3-Clause \"New\" or \"Revised\" License")
```

Comment:
This looks like it was meant to stay lowercase (as is you ended up with two identical tests)